Add two more commits from contrail_dpdk_17.02

yyang13 opened the pull request:
Hi, Raja

I checked those three commits you requested, the last one has been in dpdk 17.11, so it is unnecessary, this pull request included the former two commits, I did some changes because some among them have been in dpdk 17.11, please help merge them ASAP.

srajag commented on drivers/net/ixgbe/ixgbe_ethdev.c:
```diff
@@ -3984,8 +3984,6 @@ ixgbe_dev_link_update_share(struct rte_eth_dev *dev,
 	rte_ixgbe_dev_atomic_read_link_status(dev, &old);
 
-	hw->mac.get_link_status = true;
-
```
I think we can leave this one in for now (as there are a lot of bond driver changes in 17.11). This was added for a case where bond didn't come up with DPDK 17.02.
Rest looks ok.

ram-yadav committed:
JUMBO frame fixes in dpdk 17.02

Closes-bug: ##1703667

Signed-off-by: Yi Yang <yi.y.yang@intel.com>

srajag committed:
Enable RSS for MPLSoUDP packets on bnxt interfaces

Also set flow_type_rss_offloads so that bond interfaces will
setup RSS for UDP packets if the slave is a bnxt interface.
This reduces the likelihood of LACP flapping on the slaves.

Signed-off-by: Yi Yang <yi.y.yang@intel.com>

yyang13 commented:
I have updated this per your comment.
